crates/re_selection_panel/src/visualizer_ui.rs

```diff
@@ -1,9 +1,10 @@
 use itertools::Itertools;
 
 use re_data_ui::DataUi;
-use re_entity_db::EntityDb;
-use re_log_types::EntityPath;
+use re_entity_db::{external::re_query::LatestAtComponentResults, EntityDb};
```

Comment:
Is the `re_entity_deb::external::re_query` intentional or an IDE glitch?

Reply:
yes and no. This crate doesn't depend on re_query, but it should to make this less covert.